component/aggregated-clusterroles.libsonnet

```diff
@@ -0,0 +1,25 @@
+local kube = import 'lib/kube.libjsonnet';
+
+local cluster_reader =
+  kube.ClusterRole('syn-openshift4-monitoring-cluster-reader') {
+    metadata+: {
+      labels+: {
+        'rbac.authorization.k8s.io/aggregate-to-cluster-reader': 'true',
```

Comment:
All `monitoring.coreos.com` resources are namespaced.  Any reason to aggregate to cluster-reader instead of view?

Reply:
As we just figured out, there's a special purpose `monitoring-rules-view` clusterrole which is documented for namespaced access to user workload monitoring, so it looks like the `monitoring.coreos.com` resources were specifically excluded from `view` by RedHat. I'm not sure I agree that that's necessary, but opted for the approach which doesn't change the default behavior for end users.

Reply:
Ah right. Fair enough